NewHorizons/Assets/translations/portuguese_br.json

```diff
@@ -1,6 +1,69 @@
 {
-  "$schema": "https://raw.githubusercontent.com/xen-42/outer-wilds-new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "$schema": "https://raw.githubusercontent.com/outer-wilds-new-horizons/new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "DialogueDictionary": {
```

Comment:
Made them more star trecky

Reply:
It was just copy-pasted the english version and the schema name was like that. Felt it was better "up to date". Maybe if you want to update the schema name here, update also in the english version (and maybe another commit/pr?)
Or do you mean something else with the comment's pointed related change?

Reply:
I meant the reason behind changing to "dobra espacial" and related